### PR TITLE
Include project root directory in file path when rm-ing

### PR DESCRIPTION
**Pull request by sashton**
Fix for #3

### CODE REVIEW

File: src/leiningen/checkout/rm.clj [hunk 1]
```diff
@@ -11,21 +12,21 @@
...
-   (map (comp fs/delete (partial fs/file "checkouts")) checkouts-to-rm))
+   (map (comp fs/delete (partial fs/file root "checkouts")) checkouts-to-rm))
   checkouts-to-rm)
 
 (defn rm
   "[pattern]: Remove all checkouts. If PATTERN is specified, only checkouts matching that pattern will be removed"
-  [{:keys [checkout] :as project} & [pattern]]
-  (let [current-checkouts (fs/list-dir "checkouts")
+  [{:keys [checkout root] :as project} & [pattern]]
+  (let [current-checkouts (fs/list-dir (str root File/separator "checkouts"))
```
Comment (timvisher) [inline src/leiningen/checkout/rm.clj]: This should probably be `(fs/file root "checkouts")`

**Comment by sashton** [comment]
Thanks for the feedback. I've updated the PR to use `fs/file`.

File: src/leiningen/checkout/rm.clj [hunk 2]
```diff
@@ -1,6 +1,7 @@
 (ns leiningen.checkout.rm
   (:require [fs.core                  :as fs]
-            [leiningen.checkout.utils :as utils]))
+            [leiningen.checkout.utils :as utils])
+  (:import (java.io File)))
```
Comment (timvisher) [inline src/leiningen/checkout/rm.clj]: Is there a reason we're still importing `File`?

Comment (sashton) [inline src/leiningen/checkout/rm.clj]: Whoops, I forgot I added that. Thanks for catching it. I'll get that removed!

**Comment by sashton** [comment]
Hi @timvisher, are there any other modifications you'd like me to make for this PR?

**Comment by timvisher** [comment]
@sashton Sorry just getting back from a vacation. I'll try to get to this sometime today. :)

**Comment by timvisher** [comment]
Finally figured out the issue that I was having and documented it here https://github.com/timvisher/lein-checkout/issues/6

I should be able to verify these today and get them merged. Thanks for your patience!

**Comment by timvisher** [comment]
Alright this all looks great. Unfortunately I'm going to have to relearn how to publish to clojars. I'll do that asap.

**Pull request merged**